src/hooks/useFetch.js

```diff
@@ -1,25 +1,37 @@
-import { useState, useEffect } from 'react';
+import { useState, useEffect, useRef } from 'react';
```

Comment:
TIL `useRef` :smile:

Reply:
I learned it, too! Thanks, Megan for sharing this great [article](https://www.smashingmagazine.com/2020/07/custom-react-hook-fetch-cache-data/)!